libraries/AP_Mount/AP_Mount.h

```diff
@@ -120,6 +121,9 @@ class AP_Mount
     // used for gimbals that need to read INS data at full rate
     void update_fast();
 
+    // write mount log packet
+    void Write_Mount(enum LogMessages msg, uint64_t timestamp_us);
```

Comment:
There should be a version of Write_Mount() that takes an instance argument I think.